.rhcicd/clowdapp-engine.yaml

```diff
@@ -425,3 +427,5 @@ parameters:
   value: "false"
 - name: NOTIFICATIONS_USE_EVENT_TYPE_FOR_SUBSCRIPTION_ENABLED
   value: "false"
+- name: NOTIFICATIONS_USE_SECURED_EMAIL_TEMPLATES_ENABLED
+  value: "false"
```

Comment:
We should default to `true`  - the secure version. 
Upon merging we should add a note to set this value to false in clowdapp in the next deployment

Reply:
I undestood it's the purpose of RHCLOUD-24563. Because we already have similar flags for other features developped for secured version (like NOTIFICATIONS_EMAILS_ONLY_MODE_ENABLED), shouldn't we switch all thoses default values at once to be consistent ? WDYT ?

Reply:
for sure, RHCLOUD-24563 have to be done on next days :)

Reply:
We are already unconsistent :) some are set to secured and some are not- so I guess is better to include this one using the secured approach and migrate the rest as part of RHCLOUD-24563

Reply:
The flag is now set to `true` in the ClowdApp template. I'm resolving this conversation.